content/en/IV/datasets/cxr/_index.md

```diff
@@ -10,4 +10,126 @@ description: >
+-->
+
+## Deidentified free-text radiology reports
+
+During routine care, radiologists will review chest radiographs and document their interpretation electronically.
+When reviewing a radiograph, radiologists have access to: (1) brief text written by another clinician summarizing the underlying medical condition, (2) the reason for examination, and (3) prior imaging studies performed.
+
+Reports in MIMIC-CXR are semi-structured, and have linebreaks to ensure individual lines are no longer than 79 characters.
+As reports are templated, structure is seeded in the reports, but radiologists are free to modify it as they will before saving.
+Most reports will contain a ``FINDINGS'' and ``IMPRESSION''. The findings section details the radiologists assessment of the image, while the impression section acts as a summary of the most pertinent findings.
+
+Reports sometimes have addendums at the top. Addenums are added after the radiology report has already been written, and are intended to clarify language as necessary. Addendums are delimited from the original report by underscores which span an entire line.
+
+Radiology reports have been de-identified to protect patient privacy. All patient information has been replaced with three underscores (`___`). Provider information has also been removed.
+
+The following is an example radiology report from MIMIC-CXR:
+
+```
+                                 FINAL REPORT
+ EXAMINATION:  CHEST (PA AND LAT)
+ 
+ INDICATION:  ___ year old woman with ?pleural effusion  // ?pleural effusion
+ 
+ TECHNIQUE:  Chest PA and lateral
+ 
+ COMPARISON:  ___
+ 
+ FINDINGS: 
+ 
+ Cardiac size cannot be evaluated.  Large left pleural effusion is new.  Small
+ right effusion is new.  The upper lungs are clear.  Right lower lobe opacities
+ are better seen in prior CT.  There is no pneumothorax.  There are mild
+ degenerative changes in the thoracic spine
+ 
+ IMPRESSION: 
+ 
+ Large left pleural effusion
+```
+
+## MIMIC-CXR v2.0.0
+
+The current version of the database is v2.0.0. When referencing this version, we recommend using the full title: MIMIC-CXR v2.0.0.
+
+MIMIC-CXR v2.0.0 is the first release of the images in their native format, DICOM, and the first release of the free-text radiology reports associated with these images. All data has been de-identified prior to release to protect patient privacy.
+
+
+## Past versions
+
+#### MIMIC-CXR v1.0.0
+
+MIMIC-CXR v1.0.0 was released on 22 January 2019.
+The data contains only JPG format images and 14 structured labels extracted from an NLP tool.
+As we have reorganized the data since the release of v1.0.0, we no longer distribute v1.0.0.
+The images are identical to MIMIC-CXR v2.0.0, and we will make the structured labels available again shortly.
+
+{{% alert title="Tables" color="primary" %}}
```

Comment:
We can probably remove this alert now since this content is already in the Tables section.

Reply:
oh whoops good catch